hello-world/ACI/app/main.py

```diff
@@ -77,7 +72,7 @@ def is_hex(x): return all(c in string.hexdigits for c in x)
     # put everything together
     return (
         style +
-        "<div>" + "<h1>Welcome to Confidential containers on Azure Container Instances!</h1>" +
+        "<div>" + "<h1>Welcome to Confidential ontainers on Azure Container Instances!</h1>" +
```

Comment:
I think this should get reverted?

Reply:
woops I thought I typed the C back, good catch